### ISSUE TITLE
Hook up edit FAQ link at bottom of entry detail

### ISSUE THREAD

**Issue by sandysage** [mozilla-lockbox/lockbox-ios#172]
When viewing an entry detail, I should be informed that if I wish to edit or delete this entry, it will need to happen via Firefox desktop or mobile.

## Acceptance criteria
* Should capture user intent to manage an entry (edit, delete)
* Should inform the user that management capabilities (edit, delete) are only available in the browser for now
* Should deep link to FAQ instruction for editing both in Firefox iOS and desktop Firefox

**NOTE** This is a change from the original design. The disclaimer needs to be revised

"Learn how to edit this entry" link directs to: https://lockbox.firefox.com/faq.html#how-do-i-edit-existing-entries

## Visual Design
**InVision**
https://mozilla.invisionapp.com/share/BQEUSO0S2#/267560603_Entry_Detail

**Zeplin**
https://zpl.io/VK4vXDX

Link: https://lockbox.firefox.com/faq.html#how-do-i-edit-existing-entries

depends on #44

**Comment by sandysage** [comment]
@changecourse, @nickbrandt I believe this is still 'blocked' with needs-design.

Is the blocker here a decision on where to handle this disclaimer? Right now I see that introduced in 2 places in different forms:

1. With text at the bottom of the entry details page that reads "manage this entry using Firefox on desktop or iOS" as linked here in this issue: https://mozilla.invisionapp.com/share/BQEUSO0S2#/screens/267560603_Entry_Detail
2. With the 'edit' link up in the top which takes the user to a different screen which links off to Firefox iOS to manage that credential: https://mozilla.invisionapp.com/share/BQEUSO0S2#/screens/289206368

Or is this ready for engineering given the approach to:
1. launch with some instructions for management (perhaps with a deep link to manage in Firefox iOS)
2. introduce "edit" to capture intent
3. measure
4. introduce more editing capabilities starting with the entry name
5. measure

**Comment by nickbrandt** [comment]
@sandysage From my understanding, this is the P1 version of how we are handling editing entries with the user (providing a disclaimer on how the user can manage entries in FF Desktop/iOS). Going into P2, we begin to show the 'Edit' option (and remove this disclaimer on Entry Detail as per the design provided in https://waffle.io/mozilla-lockbox/lockbox-extension/cards/5abd22eb1c82a6001d035eee). We then show the disclaimer in the Edit mode on how they can manage their username and password, with Entry Name being the only editable field. Is there a phase I'm missing where we show the 'Edit' button but with no action? I guess I'm unsure what numbers 2-3 refer to in your comment.

**Comment by changecourse** [comment]
Updated design and ready for approval @sandysage

**Comment by sandysage** [comment]
This looks good to me. Changing ownership back to you @changecourse in support of our content review.

**Comment by sandysage** [comment]
Per IRL discussion with @changecourse @nickbrandt , we're going to update the disclaimer here to be more refined around "edit" without the caveats of "only available in Firefox desktop or Firefox for iOS"

This is intentionally structured to be a 'fake door' to capture user intent to edit (data) so we can measure the frequency of this request as well as track it relative to retention.

We're not including this as an "edit" button at the top so as to avoid deceiving users into believing we have this capability.

**Comment by changecourse** [comment]
reviewed, revised, blockers removed, ready for grooming @sandysage

**Comment by devinreams** [comment]
FAQ link is: https://lockbox.firefox.com/faq.html#how-do-i-edit-existing-entries

**Comment by sashei** [comment]
would be nice to have #44 merged before this one gets picked up.

**Comment by sashei** [comment]
related to #44
